README.md

```diff
@@ -1,2 +1,213 @@
-# web-devops-roadmap
-DevOps Roadmap for web developers
+# Web DevOps Roadmap
+
+This repository contains initial DevOps roadmap for all the web developers. Anyone can start this training with their existing learning projects.
```

Comment:
Need to mention average time taken by each exercise, so that trainee can approximate the duration.